config/default.schema.json

```diff
@@ -268,20 +268,20 @@
 		"FEATURE_ALERTS_STATUS_ENABLED": {
 			"type": "boolean",
 			"default": false,
-			"description": "If enabled, user will see messages/incidents from e.g status.hpi-schul-cloud."
+			"description": "If enabled, user will see messages/incidents from e.g status.dbildungscloud."
 		},
 		"ALERT_STATUS_URL": {
 			"type": "string",
 			"format": "uri",
 			"pattern": ".*(?<!/)$",
-			"default": "https://status.hpi-schul-cloud.de",
+			"default": "https://status.dbildungscloud.de",
 			"description": "The url of status message provider (should end without a slash)."
 		},
 		"ALERT_STATUS_API_URL": {
 			"type": "string",
 			"format": "uri",
 			"pattern": ".*(?<!/)$",
-			"default": "https://status.hpi-schul-cloud.de/api/v1",
+			"default": "https://status.dbildungscloud.de/api/v1",
```

Comment:
if this is not used anywhere in the server, and not exposed to nuxt, it can be removed.

Reply:
I can find some occurrences within server for "ALERT_STATUS_API_URL" for at least the existance of it, can it still be removed?

Reply:
looks like its being used in src/externalServices/statusApi.js
so nvm, keep it